mscore/pm.h

```diff
@@ -24,6 +24,7 @@
 #include "config.h"
 #include "mididriver.h"
 
+#include <QtGlobal>
```

Comment:
Well I'd have to change that line into #include "all.h", but I'd rather not include all.h, since it is just QtGlobal which is needed for Q_OS_WIN to be defined (needed for the line below).